File: NetBeansProjects/WebApplication1/src/java/com/epic/canvascontrollibrary/CCLLineAreaGraphProps.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */
package com.epic.canvascontrollibrary;

import java.util.ArrayList;
import java.util.List;

/**
 *
 * @author Gilgamesh
 */
    public class CCLLineAreaGraphProps
    {
        public String CanvasID;
        public String WindowID;
        public String X;
        public String Y;
        public String Width;
        public String Height;
        public List<Object> Data;
        public String XMaxValue;
        public String YMaxValue;
        public String NumMarksX;
        public String NumMarksY;
        public String Title;
        public String TitleTextColor;
        public String TitleTextHeight;
        public String TitleTextFontString;
        public String AxisLabelsColor;
        public String AxisLabelsHeight;
        public String AxisLabelsFontString;
        public String H;
        public String MarginLeft;
        public String AlreadyUnregisteredAnimation;
        public String IsLabledOnXAxis;
        public Object Tag;
        public String TabStopIndex;

        CCLLineAreaGraphProps()
        {
            Data = new ArrayList<Object>();
        }
    }

File: ASP.NET MVC Web Api New Solution/CanvasControlLibrary/CanvasControlLibrary/Models/CCLLineAreaGraphProps.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.ComponentModel.DataAnnotations.Schema;

namespace CanvasControlLibrary.Models
{
    public class CCLLineAreaGraphProps
    {
        public class LineAreaGraphData
        {
            public class DataPoint
            {
                public object XValue { get; set; }
                public List<float> YValues { get; set; }
                public DataPoint()
                {
                    YValues = new List<float>();
                }
            }
            public List<DataPoint> DataPoints { get; set; }
            public List<string> Colors { get; set; }
            public LineAreaGraphData()
            {
                DataPoints = new List<DataPoint>();
                Colors = new List<string>();
            }
        }

        public string CanvasID { get; set; }
        public int WindowID { get; set; }
        public int X { get; set; }
        public int Y { get; set; }
        public int Width { get; set; }
        public int Height { get; set; }
        public LineAreaGraphData Data { get; set; }
        public int XMaxValue { get; set; }
        public int YMaxValue { get; set; }
        public int NumMarksX { get; set; }
        public int NumMarksY { get; set; }
        public string Title { get; set; }
        public string TitleTextColor { get; set; }
        public int TitleTextHeight { get; set; }
        public string TitleTextFontString { get; set; }
        public string AxisLabelsColor { get; set; }
        public int AxisLabelsHeight { get; set; }
        public string AxisLabelsFontString { get; set; }
        public int H { get; set; }
        public int MarginLeft { get; set; }
        public int AlreadyUnregisteredAnimation { get; set; }
        public int IsLabledOnXAxis { get; set; }
        public object Tag { get; set; }
        public int TabStopIndex { get; set; }
    }
}